NewRelic.sln

```diff
@@ -13,8 +13,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Exporter.NewR
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NewRelic.Examples.AspNetCore", "examples\OpenTelemetry.Exporter.NewRelic\AspNetCore\NewRelic.Examples.AspNetCore.csproj", "{F2E815B3-5902-4DD5-9D37-CFB514D6B4DE}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NewRelic.Examples.ConsoleWithTracer", "examples\NewRelic.Telemetry\ConsoleWithTracer\NewRelic.Examples.ConsoleWithTracer.csproj", "{647540D5-0080-4594-879E-8F3394F3E391}"
```

Comment:
This design was very problematic without builders.  At this point we are minimizing use cases of the TelemetrySDK.